src/main/webapp/dev/js/common/instructorFeedbackResults.js

```diff
@@ -65,6 +65,25 @@ function showHideStats() {
     }
 }
 
+// Toggles Rubric Questions excluding self
+function switchExcludingSelfResultsForRubricStatistics(checkbox) {
+    if (checkbox.checked) {
+        $(checkbox).parent().parent().parent()
```

Comment:
don't use `parent()` chain as you implicitly enforce the DOM structure.

Reply:
Without the chained `parent()` I was unable to get the exact `.table-body-including-self` which is why I used chained parent to eventually use `find()`

Reply:
`.cloest()` method in JQuery might help you :) https://api.jquery.com/closest/